src/Forms/EmailField.php

```diff
@@ -2,52 +2,40 @@
 
 namespace SilverStripe\Forms;
 
+use SilverStripe\Core\Validation\ConstraintValidator;
+use Symfony\Component\Validator\Constraints\Email as EmailConstraint;
+
 /**
- * Text input field with validation for correct email format according to RFC 2822.
+ * Text input field with validation for correct email format according to the relevant RFC.
```

Comment:
The listed RFC was out of date. Better to just not specify the RFC especially now we've offloaded the actual validation to Symfony who will no doubt update their validation to new RFCs as they become available.